tests/echoes/test_full_patch.py

```diff
@@ -0,0 +1,25 @@
+from open_prime_rando import echoes_patcher
+
+
+def test_ntsc_paks(prime2_iso_provider, tmp_path, test_files_dir):
+    output_path = tmp_path.joinpath("out")
+    configuration = test_files_dir.read_json("echoes", "door_lock.json")
+
+    echoes_patcher.patch_paks(
+        file_provider=prime2_iso_provider,
+        output_path=output_path,
+        configuration=configuration,
+    )
+    assert len(list(output_path.rglob("*.pak"))) == 7
+
+
+def test_pal_paks(pal_prime2_iso_provider, tmp_path, test_files_dir):
```

Comment:
is it possible to parameterize which iso provider is used? would be nice since these functions are otherwise identical, but it's not a huge deal

Reply:
ehh maybe? would require some stuff because fixtures with params is a bit different.
Though I'm not sure I want to run all the tests with both ISOs, I'm happy with just this one test for PAL iso

Reply:
ah yeah I meant parametrizing this specific test, not the fixtures themselves. nbd if that's a pain